metacatalog/models/variable.py

```diff
@@ -165,5 +165,71 @@ def to_dict(self, deep=False) -> dict:
 
         return d
 
+    @classmethod
+    def from_dict(cls, data: dict, session: Session) -> 'Variable':
```

Comment:
Not possible to test this, as I cannot add a unit to the variable in the Metacatalog Entry Creator.